libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -544,7 +544,8 @@ bool SRV_Channels::set_aux_channel_default(SRV_Channel::Aux_servo_function_t fun
         // already assigned
         return true;
     }
-    if (channels[channel].function != SRV_Channel::k_none) {
+    if (channels[channel].function != SRV_Channel::k_none &&
+        !(channel >15 && channels[channel].function == SRV_Channel::k_GPIO)) {
```

Comment:
This is because if this I guess.

https://github.com/ArduPilot/ardupilot/blob/0c9aaa2ae6b9b70fbf132f733c51bd90d903d56b/libraries/SRV_Channel/SRV_Channels.cpp#L355-L360

Which is in turn because of this:

https://github.com/ArduPilot/ardupilot/blob/0c9aaa2ae6b9b70fbf132f733c51bd90d903d56b/libraries/AP_Logger/LogFile.cpp#L189

Its a bit odd that we have to log the None output type, we really should move them over to a dedicated "servo events function" so none really means disabled.